mod.ts

```diff
@@ -38,11 +39,27 @@ export async function handler(request: Request) {
       }],
     },
     options: {
+      color: dark ? "white" : undefined,
       scales: {
         y: {
           title: {
             display: true,
             text: "avg. ns/iter",
+            color: dark ? "white" : undefined,
+          },
+          grid: {
+            color: dark ? "rgba(255,255,255,0.5)" : undefined,
+          },
+          ticks: {
+            color: dark ? "white" : undefined,
+          },
+        },
+        x: {
+          grid: {
+            color: dark ? "rgba(255,255,255,0.5)" : undefined,
+          },
+          ticks: {
+            color: dark ? "white" : undefined,
```

Comment:
It looks fine on my computer? I'll set the zero line color as well just in case.

<img width="724" alt="image" src="https://user-images.githubusercontent.com/11367844/234416790-91b3e789-24d7-4b37-855d-ceb84846d54d.png">